book/src/getting-started.md

```diff
@@ -39,24 +35,14 @@ It is also possible to specify other users with the configuration file.
 
 ```bash
 cat <<EOF | adduser -w none -f -
-pjdfs_dummy_1::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
-pjdfs_dummy_2::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
-pjdfs_dummy_3::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
-pjdfs_dummy_4::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
-pjdfs_dummy_5::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
-pjdfs_dummy_6::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
+tests::::::Dummy User for pjdfstest:/nonexistent:/sbin/nologin:
```

Comment:
Actually, this user already exists, too, ever since Kyua got imported into the base system in 2020.

Reply:
Hmm, that's why the CI fails!